docs/src/api/class-eventlist.md

```diff
@@ -0,0 +1,70 @@
+# class: EventList
+* langs: js
+* since: v1.28
+
+Event list collects events of a certain kind for future use; instances
+of this class are accessible via the [`property: Page.events`] getter.
+
+Notice that event lists simply store all historic events of a given type and
+might store outdated objects, e.g. `page.events.popup.all()` might return
+already-closed pages.
+
+Use event lists instead of a `Promise.all` pattern. Consider the following
+example:
+
+```js
+// Before:
+const [consoleMessage] = await Promise.all([
+  page.waitForEvent('console'),
+  page.getByText('Log Console Message').click(),
+]);
+
+// After:
+page.events.console.track();
```

Comment:
> Make separate js-ts vs js-library snippets, and omit track() in the testing snippet.

This would make sense once we engage in playwright test by default